ProjectName/src/store/monthTabSlice/index.ts

```diff
@@ -0,0 +1,12 @@
+import {useAppDispatch, useAppSelector} from '../hooks';
+import {changeMonth, monthSelector} from './monthTabSlice';
+
+export const useMonthTab = () => {
+  const monthState = useAppSelector(monthSelector);
+  const dispatch = useAppDispatch();
+  const changeMonthAction = dispatch(changeMonth);
+  return {
```

Comment:
In case your function is more than 3 lines, please separate `return` and rest of the function body with blank line.